pandas/core/internals/array_manager.py

```diff
@@ -454,7 +471,7 @@ def is_mixed_type(self) -> bool:
 
     @property
     def is_numeric_mixed_type(self) -> bool:
-        return False
+        return all(is_numeric_dtype(t) for t in self.get_dtypes())
```

Comment:
should the "mixed" be meaningful here, i.e. what if we're homogeneous dtype?

Reply:
Then it will also return True, like it does for BlockManager (this basically only wants to know that all columns are numeric, while not necessarily being all the same dtype)